source

```diff
@@ -100249,6 +100249,19 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
      </div>
 
     <tr>
+     <td><dfn data-x="parse-error-unexpected-doctype">unexpected-doctype</dfn>
+     <td>
+      <p>This error occurs if the parser encounters an incorrectly placed <span
```

Comment:
It would be nice to elaborate what we mean by "incorrectly placed": if DOCTYPE is not a document preamble, i.e. there is content that is not comments or ASCII whitespace before it.